### PR TITLE
New Format for Report

### PR DESCRIPTION
**Pull request by seizadi**

### CODE REVIEW

**Comment by seizadi** [comment]
Various changes and support for python3

File: epics_export.py [hunk 1]
```diff
@@ -55,23 +55,23 @@ def query(scope, filter, debug=False):
...
         else:
-            print ("%s, %s" % (s['Number'],s['Name']))
+            print("%s, %s" % (s['Number'],s['Name']))
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description='export versionone stories.')
     parser.add_argument('--token', default=os.environ.get('VERSION_ONE_TOKEN'))
     parser.add_argument('--endpoint', default=os.environ.get('VERSION_ONE_ENDPOINT'))
-    parser.add_argument("--scope", default="Athena 2.7")
+    parser.add_argument("--scope", default="Salus 2.4")
```
Comment (chipblox) [inline epics_export.py]: This should be Portunus 5.2

**Comment by seizadi** [comment]
Changes for new format...

**Comment by seizadi** [comment]
I rebased, the major changes were:
- Adding support for rejected stories
- Breaking out DA Review from Re-review

**Pull request merged**